marketplace/deployer_helm_tiller_base/test/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG MARKETPLACE_TOOLS_TAG
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_helm_tiller:$MARKETPLACE_TOOLS_TAG
+
+COPY marketplace/deployer_helm_tiller_base/test/schema.yaml /data
+
+COPY marketplace/deployer_helm_tiller_base/test/chartmuseum-marketplace /tmp/chartmuseum-marketplace
+
+RUN mkdir -p /data/chart \
+ && cd /data/chart \
+ && helm dependency build /tmp/chartmuseum-marketplace \
```

Comment:
I thought more about this and we really shouldn't dictate how the partner builds the helm chart. This command shouldn't be part of ONBUILD.

The CI test can run `helm dependency build` to build out the chart directory. The test would then also run docker build, etc.

For future follow up.